suse_migration_services/units/prepare.py

```diff
@@ -115,9 +116,18 @@ def main():
     zypp_plugins_services = os.sep.join(
         [root_path, 'usr', 'lib', 'zypp', 'plugins', 'services']
     )
-    cloud_register_metadata = os.sep.join(
-        [root_path, 'var', 'lib', 'cloudregister']
-    )
+
+    with open(hosts_setup, 'r', encoding="utf-8") as fp:
+        if 'susecloud.net' in fp.read():
+            try:
+                cloud_register_metadata = \
+                    get_regionsrv_client_file_location(root_path)
+            except DistMigrationZypperMetaDataException as issue:
+                log.error('Failed locating regionsrv-client cache files: {0}'.format(
```

Comment:
just a bit nitpicking, maybe consider
```python
              log.error(
                    'Failed locating regionsrv-client cache files: {0}'.format(
                          issue
                    )
              )
```
so you keep closing brackets consistently

Reply:
no problem